h/migrations/versions/ff02f2c58_add_the_organization_table.py

```diff
@@ -0,0 +1,32 @@
+"""
+Add the organization table.
+
+Revision ID: ff02f2c58
+Revises: 7bf167611bc3
+Create Date: 2018-03-15 11:00:50.420618
+"""
+
+from __future__ import unicode_literals
+
+from alembic import op
+import sqlalchemy as sa
+
+revision = 'ff02f2c58'
+down_revision = '7bf167611bc3'
+
+
+def upgrade():
+    op.create_table(
+        'organization',
+        sa.Column('id', sa.Integer, autoincrement=True, primary_key=True),
+        sa.Column('created', sa.DateTime(), server_default=sa.func.now(), nullable=False),
+        sa.Column('updated', sa.DateTime(), server_default=sa.func.now(), nullable=False),
```

Comment:
I don't think it makes a difference but we should be consistent about whether we pass classes or instances to `sa.Column()` - `sa.DateTime` vs `sa.DateTime()`. I think classes rather than instances are slightly more common in the codebase so far and would be consistent with the other columns in this file.